www/src/components/tech-with-icon.js

```diff
@@ -1,23 +1,20 @@
-import React from "react"
-
-const TechWithIcon = ({ icon, height, children }) => {
-  let h = height ? height : `1.2em`
+/** @jsx jsx */
+import { jsx } from "theme-ui"
 
+export default function TechWithIcon({ icon, height = `1.2em`, children }) {
   return (
-    <span css={{ whiteSpace: `nowrap` }}>
+    <span sx={{ whiteSpace: `nowrap` }}>
       {children}
       &nbsp;
       <img
         src={icon}
-        alt=""
-        css={{
-          height: `${h}`,
-          margin: 0,
+        alt={children}
```

Comment:
This isn't right. In this case, the icon is purely decorative and has no context. Screenreaders will read this as "React react" 

```suggestion
        alt=""
```

Reply:
Ohh, I was not knowing about that, thank you!